shell/platform/darwin/common/framework/Headers/FlutterCodecs.h

```diff
@@ -249,10 +249,17 @@ FLUTTER_EXPORT
  * @param code An error code string for programmatic use.
  * @param message A human-readable error message.
  * @param details Custom error details.
+ * @param stacktrace Native stacktrace for the error.
  */
 + (instancetype)errorWithCode:(NSString*)code
                       message:(NSString* _Nullable)message
                       details:(id _Nullable)details;
+
++ (instancetype)errorWithStacktrace:(NSString*)code
```

Comment:
This is not correct ObjC naming; this should still be `errorWithCode:` since the thing right after it is the code.